Documentation/Signoffs/power-distribution-signoff.md

```diff
@@ -0,0 +1,223 @@
+	+ This constraint was made because the purpose of the power distribution system is to provide adequate power the entire robot. Because of this, the other subsystems constrain the power distribution subsystem and this constraint is the most important to this subsystem.
+- C4: The robot shall contain modules that are plug-and-play
+adaptable for different IEEE competition requirements.
+	+ This constraint is applicable to all subsystems of the robot platform in order to make the robot easily adaptable year to year. This in particular applies to the power distribution system by requiring that power connections should be easy to connect and disconnect.
+- C7:  The robot shall possess a single emergency stop button
+that is accessible and stops all robot movement without removing power to essential processors that is compliant with standard NFPA 79-10 [1].
+	+ The constraint is required by the IEEE hardware competition rules every year as a safety precaution and is followed by the power distribution system of the robot [2][3][4].
+- C8: The robot shall contain a power bus that is not inhibited by DC motor operation.
+	+ This constraint was made because DC motors can cause interference in the power buses of electronics by supplying a voltage pushing back on their power source, referred to as "back-EMF", inhibiting the function of the power rails for other components. Diode protection is used to prevent this.
+
+### Derived from Broader Implications
+
+- C17: This standard outlines the allowable ampacities of insulated conductors up to 2000 volts. The project will be well under this voltage and is therefore applicable. The standard will need to be followed for the safety of the design team and potential users by ensuring all wire is appropriately rated for the load [5].
+	+ This constraint is applicable because it outlines the allowable ampacities of insulated conductors up to 2000 Volts. The voltages used in the power distribution system will fall under this limit and so this standard is an applicable constraint of the system.
+
+## Schematic
+
+![Circuit Board Schematic](./Images/power_dist_schematic.png)
+
+The schematic for the power distribution circuit board is shown above. The board has a screw terminal connector meant for connecting to the battery. This then supplies the input voltage to the three power rail subcircuits. 
+
+The circuit board is based around a single type of buck converter chip, the TPS565201. This chip is a variable voltage regulator buck converter that can source up to 5 A of output current [6]. The output voltage is set by the ratio of feedback resistors connected to the the VFB pin of the chip. 
+
+There are two buck converter-based subcircuits on the board, one for the 3.3 V and one for the 5 V rail. The feedback resistors are R5, R7, R4, and R6, which set output voltages of 5 V and 3.3 V, respectively.
```

Comment:
Do you need to calculate the feedback resistance value?

Reply:
The datasheet provides a formula for calculating the feedback resistance values by hand but also includes a table of common output voltage and their respective component values. I used the recommended values (or the nearest standard value) for both the 5V and 3.3V regulator circuits.

Reply:
My update should hopefully clarify this point.